src-docs/src/views/toast/guidelines.js

```diff
@@ -75,6 +76,12 @@ and space to read it properly. Alternatively just link to a full page.
           They are a popular design choice because they don&apos;t need to fit
           in a layout and don&apos;t disrupt the user.
         </p>
+
+        <p>
+          Default toast messages appear, give users time to read or listen to
+          the announcement, and fade out. They communicate passive information
+          that does not require the user to take action.
```

Comment:
I'm not sure we can add this prescriptive guideline without adjusting our own examples later in the page:

<img width="500" alt="" src="https://user-images.githubusercontent.com/549407/228925359-cdcc0df6-9dc3-4f5d-b2bc-c84fba32e79f.png">

I think a broader approach may be something like this:

```suggestion
          the announcement, and fade out. They generally communicate passive information
          that does not require the user to take action. If user action is required, focus state must be manually managed in order for keyboard users to have immediate access to the action.
```